src/assets/Email.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="h-6 w-6" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Change all `class=` to `className=` since only `class=` will cause error log on console. I see 6 of these.
```suggestion
<svg className="h-6 w-6" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```